hw/ip/pattgen/data/pattgen_testplan.hjson

```diff
@@ -266,6 +222,12 @@
             Similar to pattern_data_ch0_1_cg.
             '''
     }
+    {
+      name: program_while_busy_cg
+      desc: '''
+            Program the DUT while pattern is being produced to insure current pattern is not being corrupt.
```

Comment:
does any test have this sequence? if no, can you the sequence to the test or create a new one?

Reply:
@weicaiyang IMO creating this sequence is a V2 task. the V1 task is to plan the V2 work with the needed testpoints
which alex has done here :D

Reply:
I actually mean to add a sequence to existing test entry in the testplan or create a new test entry for this.